src/components/LandingPage/index.js

```diff
@@ -5,6 +5,7 @@ import NewHeader from "../NewHeader";
 import LandingFooter from "../LandingFooter";
 import { ReactComponent as InfrastrucurePic } from "../../assets/images/infrastructure.svg";
 import { ReactComponent as RightArrow } from "../../assets/images/rightArrow.svg";
+import { ReactComponent as RightArrowGray } from "../../assets/images/right-arrow-gray.svg";
```

Comment:
@fel-w we decided against the parts that have these arrows because they currently do not lead anywhere, could you kindly remove them. The arrows and the components thereof, like learn more and the others.

Reply:
![rq-removed arrors and hover](https://user-images.githubusercontent.com/63339234/124921546-3002a180-e001-11eb-8a0c-9d6ca8c3a6ff.png)

hope this solves the issue!